authorization/auth_cloudSolutionProvider.md

```diff
@@ -0,0 +1,83 @@
+# Call Microsoft Graph from a Cloud Solution Provider application
+
+> **Note:** This topic applies **only** to Microsoft Cloud Solution Provider (CSP) application developers. The <a href="https://partner.microsoft.com/en-US/cloud-solution-provider" target="_newtab">Microsoft Cloud Solution Provider (CSP)</a> program enables Microsoft’s partners to resell and manage Microsoft Online services to customers. 
+
+This topic describes how to enable application access to partner-managed customer data via Microsoft Graph using either the <a href="https://docs.microsoft.com/en-us/azure/active-directory/develop/active-directory-protocols-oauth-code" target="_newtab">authorization code grant flow</a> or the <a href="https://docs.microsoft.com/en-us/azure/active-directory/develop/active-directory-protocols-oauth-service-to-service" target="_newtab">service to service client credentials flow</a>. 
+
+
+**Important:** Calling Microsoft Graph from a CSP application is only supported for directory resources (such as **user**, **group**,**device**, **organization**) and [Intune](..\beta\resources\intune_graph_overview) resources.
+
+## What is a Partner-Managed Application?
+The CSP program enables Microsoft’s partners to resell and manage Microsoft Online Services (such as Office 365, Microsoft Azure, and CRM Online) to customers. Management of customer services is done through Delegated Admin Privileges, which enables designated partner users and agents to access and configure their customers’ environments.
+
+Additionally, as a partner developer, you can build a Partner-Managed App (PMA), also known as a *pre-consented* app, to manage your customers' Microsoft services. PMAs are often termed *pre-consented* apps because all your customers are automatically pre-consented for your partner-managed apps. This means when a user from one of your customer tenants uses one of your partner-managed apps, the user can use it without seeing any consent. And with the Delegated Admin Privilegesmodel, your partner agents can access customers through your partner-managed application.
```

Comment:
Dan, please correct me if I am totally off, but should this phrase, "... termed as pre-consented apps because all your customers are automatically pre-consented for your partner-managed apps" be "termed as pre-consented apps because all your customers automatically give consent to your partner-managed apps"?

If I didn't get it wrong, then the phrase "... the user can use it without seeing any consent" should be "... the user can use it without being prompted to give consent"? After all, how does one "see consent"?